bundles/action/org.openhab.action.telegram/src/main/java/org/openhab/action/telegram/internal/Telegram.java

```diff
@@ -66,10 +66,14 @@
 
     private static Map<String, TelegramBot> groupTokens = new HashMap<String, TelegramBot>();
 
-    public static void addToken(String group, String chatId, String token, String parse) {
-        groupTokens.put(group, new TelegramBot(chatId, token, parse));
+    public static void addToken(String group, String chatId, String token) {
+        groupTokens.put(group, new TelegramBot(chatId, token));
     }
 
+    public static void addToken(String group, String parse) {
```

Comment:
I don't believe this method does anything. The bot created by this method will never function, will it?